Paws4Me/Paws4Me/Config/AppDelegate.swift

```diff
@@ -6,23 +6,63 @@
 //
 
 import UIKit
+import CoreData
 
 @main
 class AppDelegate: UIResponder, UIApplicationDelegate {
 
-    func application(_ application: UIApplication,
-                     didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
-        return true
-    }
+  var window: UIWindow?
 
-    // MARK: UISceneSession Lifecycle
+  func application(_ application: UIApplication,
+                   didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+    return true
+  }
 
-    func application(_ application: UIApplication,
-                     configurationForConnecting connectingSceneSession: UISceneSession,
-                     options: UIScene.ConnectionOptions) -> UISceneConfiguration {
-        return UISceneConfiguration(name: "Default Configuration", sessionRole: connectingSceneSession.role)
-    }
+  func applicationWillTerminate(_ application: UIApplication) {
+    self.saveContext()
+  }
+
+  // MARK: - Core Data stack
+  lazy var persistentContainer: NSPersistentContainer = {
+    // The persistent container for the application. This implementation
```

Comment:
You can remove all these generated comments.

Reply:
Done 👍🏻 I originally left it in for understanding purposes.